src/OKLogger/Parsing/ValueEscaper.cs

```diff
@@ -23,9 +23,8 @@ public string Escape(string value)
             }
             foreach(var c in EscapedChars)
             {
-                value = value.Replace(c, string.Empty);
+                value = value.Replace(c, '\0'); // \0 is an ASCII value of 0 aka no value
```

Comment:
I'm not sure how various programs will handle the \0.

Why not just replace " (double-quote) with ' (single quote). The need is quite specific - we don't require a generic escaping class.